packages/transforms/mocking/yaml-config.graphql

```diff
@@ -0,0 +1,44 @@
+extend type Transform {
+    """
+    Mocking configuration for your source
+    """
+    mocking: MockingConfig
+}
+
+type MockingConfig {
+    """
+    If this expression is truthy, mocking would be enabled
+    """
+    if: Boolean
+    """
+    Do not mock any other resolvers other than defined in `mocks`
+    """
+    preserveResolvers: Boolean
+    """
+    Mock configurations
+    """
+    mocks: [MockingFieldConfig]
+}
+
+type MockingFieldConfig {
+    """
+    Resolver path
+    Example: User.firstName 
+    """
+    apply: String!
+    """
+    If this expression is truthy, mocking would be enabled
+    """
+    if: Boolean
+    """
+    Faker.js expression
+    Read more (https://github.com/marak/Faker.js/#fakerfake)
+    """
+    faker: String
```

Comment:
Maybe this should be either a function name, OR an expression? 
Something like :

```
faker: name.firstName
faker: "Test {{ name.firstName }}"
```

Because using `{{ .. }}` all the time seems a bit annoying

Reply:
Done!